### PR TITLE
disable altruistic referral incentive for squads

### PR DESCRIPTION
**Pull request by amaliwanag2**
already changed the messaging on the home page

### CODE REVIEW

File: graphql/database/users/__tests__/logEmailVerified.test.js [hunk 1]
```diff
@@ -145,7 +145,42 @@ describe('logEmailVerified', () => {
...
+      Attributes: expectedReturnedUser,
+    })
+
+    setMockDBResponse(DatabaseOperation.QUERY, {
+      Items: [{ inviterId: 'someInviterId', invitedEmail: 'foo@bar.com' }],
+    })
+    const updateSpy = jest.spyOn(InvitedUsersModel, 'update')
+    const logEmailVerified = require('../logEmailVerified').default
+    await logEmailVerified(modifiedUserContext, modifiedUserContext.id)
+    expect(rewardReferringUser).toHaveBeenCalledWith(
```
Comment (kmjennison) [inline graphql/database/users/__tests__/logEmailVerified.test.js:180]: Should this be `.not` called?

File: graphql/database/users/__tests__/rewardReferringUser.test.js [hunk 2]
```diff
@@ -675,6 +675,72 @@ describe('rewardReferringUser', () => {
...
+    setMockDBResponse(DatabaseOperation.UPDATE, {
+      Item: {
+        ...getMockUserImpact(),
+        userId,
+      },
+    })
+    const updateMethod = jest.spyOn(UserImpactModel, 'update')
+    await rewardReferringUser(userContext, userId, '123456789')
+    expect(updateMethod).not.toHaveBeenCalled()
+    expect(updateMethod).not.toHaveBeenCalled()
```
Comment (kmjennison) [inline graphql/database/users/__tests__/rewardReferringUser.test.js:741]: Remove duplicate `expect`?

**Review by kmjennison — CHANGES_REQUESTED** [review]
Just a few questions (I might be missing context). Otherwise, LGTM.

File: graphql/database/users/__tests__/logEmailVerified.test.js (hunk 1 above)
Comment (amaliwanag2) [reply graphql/database/users/__tests__/logEmailVerified.test.js:180]: yeah lol i should just not be calling this function

**Pull request closed without merging**